server/main.py

```diff
@@ -178,23 +178,21 @@ def cameras():
         camera.mode = camera.mode.value
     return render_template("cameras.html", cameras=cameras)
 
-@bp.route("/train")
+@bp.route("/train", methods=['GET', 'POST'])
 @login_required
 def train():
+    if request.method == "POST":
+        file = request.files['file']
+        if file.filename != '':
+            if file:
+                file.filename = secure_filename(file.filename)
+                output = send_to_s3(file, "lfiasimagestore")
+                return str(output)
+        else:
+            return redirect("myapp.home")
+    
```

Comment:
I combined the two separate `/train` API routes and consolidated into just one function.